bin/i18n/resources/dashboard/blocks/sync_in.rb

```diff
@@ -39,24 +36,8 @@ def process
             end
           end
 
-          private def i18n_data
-            {
-              'en' => {
-                'data' => {
-                  'blocks' => blocks_data
-                }
-              }
-            }
-          end
-
           private def prepare
-            I18nScriptUtils.write_yaml_file(ORIGIN_I18N_FILE_PATH, i18n_data)
-            I18nScriptUtils.copy_file(ORIGIN_I18N_FILE_PATH, I18N_SOURCE_FILE_PATH)
-          end
-
-          private def redact
-            I18nScriptUtils.copy_file(I18N_SOURCE_FILE_PATH, I18N_BACKUP_FILE_PATH)
-            RedactRestoreUtils.redact(I18N_SOURCE_FILE_PATH, I18N_SOURCE_FILE_PATH, REDACT_PLUGINS, REDACT_FORMAT)
+            I18nScriptUtils.write_json_file(I18N_SOURCE_FILE_PATH, blocks_data)
```

Comment:
Yes, correct.
However, we will need to migrate yml files to json files eventually.
And the inconvenience now of having to re-do translations will save many errors in the future.
The international team is revisiting all blocks and updating many translations at this time, so it would be ideal to make the JSON file available ASAP. 
Meanwhile, we would keep the existing YAML file translations, until the JSON file is fully translated.

Reply:
I completely agree with this; I'm only concerned that this could potentially be the cause of the bug reports regarding missing block translations

Reply:
We will keep old translations until the international team lets us know that the JSON files are fully translated. Then, I have a second PR removing the translated YML files.